### PR TITLE
feat: show time in documents table

### LINKED ISSUE #1077: Adding time to the document table
### Feature Description

I suggest adding the time information to the document table.

image

### Use Case

_No response_

### Proposed Solution

_No response_

### Alternatives (optional)

_No response_

### Additional Context

_No response_

### Please check the boxes that apply to this feature request.

- [X] I have searched the existing feature requests to make sure this is not a duplicate.
- [ ] I have provided a detailed description of the requested feature.
- [ ] I have explained the use case or scenario for this feature.
- [ ] I have included any relevant technical details or design suggestions.
- [ ] I understand that this is a suggestion and that there is no guarantee of implementation.
- [ ] I want to work on creating a PR for this issue if approved

### PR DESCRIPTION
**Pull request by rushatgabhane**
---
name: Pull Request
about: Submit changes to the project for review and inclusion
---

## Description

Display time in 12h format in the documents table.

## Related Issue

Fixes #1077 
## Changes Made

- Use DateTime.DATETIME_SHORT
- ...

## Testing Performed

- Tested time in different timezone.
- Ran tests in web and mobile browser

1. Login 
2. Add a document
3. Verify that you see date and time in 12h format as per your locale.

## Checklist

- [x] I have tested these changes locally and they work as expected.
- [x] I have added/updated tests that prove the effectiveness of these changes.
- [x] I have updated the documentation to reflect these changes, if applicable.
- [x] I have followed the project's coding style guidelines.
- [x] I have addressed the code review feedback from the previous submission, if applicable.

## Additional Notes
Screenshot 2024-04-29 at 04 21 53

Screenshot 2024-04-29 at 04 22 18

### CODE REVIEW

**Comment by Mythie** [comment]
Hey there, it looks like you haven't accepted our contributor license agreement yet. In order for us to accept your pull request we ask that you please fill out the CLA:

https://documen.so/cla

**Comment by rushatgabhane** [comment]
CLA signed. Please re-run

File: apps/web/src/app/(dashboard)/documents/data-table.tsx [hunk 1]
```diff
@@ -62,7 +63,9 @@ export const DocumentsDataTable = ({
           {
             header: 'Created',
             accessorKey: 'createdAt',
-            cell: ({ row }) => <LocaleDate date={row.original.createdAt} />,
+            cell: ({ row }) => (
+              <LocaleDate date={row.original.createdAt} format={DateTime.DATETIME_SHORT} />
```
Comment (adithyaakrishna) [inline apps/web/src/app/(dashboard)/documents/data-table.tsx:67]: Tested it locally and LGTM

- Date comes in 12hr format 
- Is in line with our Timestamp design according to our design file

Screenshot 2024-05-08 at 3 23 38 PM

**Review by adithyaakrishna — APPROVED** [review]

**Comment by ElTimuro** [comment]
@rushatgabhane For Germany, I'm getting a 24-hour format. Is this by design, since Germany does not have a 12-hour format?
image

**Comment by adithyaakrishna** [comment]
@ElTimuro AFAIK this might be the luxon library doing its thing, we can also have a specific format `MM/dd/yyyy, hh:mm a` for it

**Comment by ElTimuro** [comment]
Since we are not localizing yet, we should keep it consistent, I think

**Comment by rushatgabhane** [comment]
Makes sense. Let me make the change to be 12h

**Comment by rushatgabhane** [comment]
@ElTimuro done! Ready for re-review @adithyaakrishna

**Review by adithyaakrishna — APPROVED** [review]
Checked it and works fine now :)

@eltimuro, could you pls confirm the same as well?

Screenshot 2024-05-08 at 9 06 44 PM

**Review by ElTimuro — APPROVED** [review]
- Checked locally
- Looks good, merging

**Pull request merged**